Expected stderr:

▼ connect X.Xs
├╴✔ starting engine X.Xs
├╴▼ connecting to engine X.Xs
│ ┃ XX:XX:XX INF connected name=xxxxxxxxxxxxx.xxxxxxxxxxxxx.dagger.local client-version=vX.X.X-xxxxxxxxxxxx-xxxxxxxxxxxx server-version=vX.X.X-xxxxxxxxxxxx-xxxxxxxxxxxx
╰╴✔ starting session X.Xs
 
▼ load module: ./viztest/typescript X.Xs
├╴✔ finding module configuration X.Xs
├╴✔ initializing module X.Xs
├╴✔ inspecting module metadata X.Xs
╰╴✔ loading type definitions X.Xs
 
✔ parsing command line arguments X.Xs

✔ typescript: Typescript! X.Xs
▼ .failEffect: Container! X.Xs ERROR
! process "sh -c echo this is a failing effect; exit 1" did not complete successfully: exit code: 1
├╴$ Container.from(address: "alpine"): Container! X.Xs CACHED
╰╴▼ .withExec(args: ["sh", "-c", "echo this is a failing effect; exit 1"]): Container! X.Xs ERROR
  ┃ this is a failing effect
  ! process "sh -c echo this is a failing effect; exit 1" did not complete successfully: exit code: 1

Error logs:

▼ .withExec(args: ["sh", "-c", "echo this is a failing effect; exit 1"]): Container! X.Xs ERROR
this is a failing effect
! process "sh -c echo this is a failing effect; exit 1" did not complete successfully: exit code: 1

Setup tracing at https://dagger.cloud/traces/setup. To hide set DAGGER_NO_NAG=1
